### PR TITLE
Add RH Cloud and virt-who Configure to Katello Devel Role

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: roles/katello_devel/meta/main.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+ca---
```
Comment (ianballou) [inline roles/katello_devel/meta/main.yml:1]: :canada: 

Typo? :)

Comment (chris1984) [reply roles/katello_devel/meta/main.yml:1]: Fat fingers, let me fix it

Comment (chris1984) [reply roles/katello_devel/meta/main.yml]: @ianballou fixed

**Comment by chris1984** [comment]
@ehelms can this get a merge, simple one :)

**Pull request merged**